components/Placeholder/Placeholder.styled.js

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+
+export const Placeholder = styled.div`
```

Comment:
What is the purpose of this component? Is it to make sure that the page is at least as tall as the viewport? 

If so, i think there are better ways to achieve this. No requiring any changes this time. Just something to think about.

Reply:
the placeholder goes to the bottom of the pages to make sure the last event on the page is not covered by the NavigationBar